tests/scripts/turboquant4_hf_cache_eval.py

```
@@ -0,0 +1,571 @@
+        decode_nll_per_token=(
+            decode_nll / decode_token_count if decode_token_count > 0 else None),
+    )
+
+
+def _compare_outputs(
+    prompts: list[EvalPrompt],
+    baseline_outputs: list[OutputRecord],
+    turbo_outputs: list[OutputRecord],
+) -> tuple[list[PromptRecord], dict[str, Any]]:
+    records = []
+    for prompt, baseline, turbo in zip(prompts, baseline_outputs, turbo_outputs):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)